### ISSUE TITLE
decompile (java.lang.IndexOutOfBoundsException)

### ISSUE THREAD

**Issue by deemru** [wavesplatform/Waves#3564]
**Describe the bug**
Unable to decompile the script [9hbRAKaRUEufHDtYuiH4k4s8dWWp11ermmKcoNLKGtN5](https://w8io.ru/tx/9hbRAKaRUEufHDtYuiH4k4s8dWWp11ermmKcoNLKGtN5) (MAINNET)

```java
ERROR [avy-request-processor-5-3] c.w.api.http.package$ - Uncaught error
java.lang.IndexOutOfBoundsException: 1
        at scala.collection.LinearSeqOps.apply(LinearSeq.scala:117)
        at scala.collection.LinearSeqOps.apply$(LinearSeq.scala:114)
        at com.wavesplatform.lang.v1.compiler.Decompiler$.$anonfun$expr$3(Decompiler.scala:191)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.expr(Decompiler.scala:127)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.$anonfun$expr$9(Decompiler.scala:168)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.expr(Decompiler.scala:127)
        at sequence @ com.wavesplatform.lang.v1.Serde$.$anonfun$desAuxR$10(Serde.scala:118)
        at value @ com.wavesplatform.lang.v1.Serde$.$anonfun$deserialize$5(Serde.scala:175)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.expr(Decompiler.scala:127)
        at sequence @ com.wavesplatform.lang.v1.Serde$.$anonfun$desAuxR$10(Serde.scala:118)
        at sequence @ com.wavesplatform.lang.v1.Serde$.$anonfun$desAuxR$10(Serde.scala:118)
        at sequence @ com.wavesplatform.lang.v1.compiler.Decompiler$.intersperse$1(Decompiler.scala:253)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.$anonfun$expr$3(Decompiler.scala:196)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.expr(Decompiler.scala:127)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.$anonfun$expr$3(Decompiler.scala:167)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.expr(Decompiler.scala:127)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.$anonfun$decl$1(Decompiler.scala:39)
        at cached @ com.wavesplatform.lang.v1.compiler.Decompiler$.decl(Decompiler.scala:36)
        at sequence @ com.wavesplatform.lang.v1.Serde$.$anonfun$desAuxR$10(Serde.scala:118)
```

**To Reproduce**
1. http://nodes.wavesnodes.com/api-docs/index.html#/utils/decompileScript
2. paste script from [9hbRAKaRUEufHDtYuiH4k4s8dWWp11ermmKcoNLKGtN5](https://w8io.ru/tx/9hbRAKaRUEufHDtYuiH4k4s8dWWp11ermmKcoNLKGtN5) (MAINNET)
3. get error

**Expected behavior**
No error on decompile.

**Additional context**
1.4.0 still cannot decompile

**Comment by phearnot** [comment]
Fixed in #3559 and will be released as part of 1.3.9.

**Comment by deemru** [comment]
Fixed, thanks.
